.github/workflows/workflow.yml

```diff
@@ -148,7 +148,7 @@ jobs:
        uses: actions/cache@v2
        with: 
         path: tmp/
-        key: OpenHAB-setup-${{ hashFiles('.bundlehash', 'Rakefile') }}-${{ matrix.openhab_version }}
+        key: OpenHAB-setup-${{ hashFiles('.bundlehash', 'Rakefile', 'rakelib/openhab.rake') }}-${{ matrix.openhab_version }}
```

Comment:
can I get a little help here? it seems it's still using the key `OpenHAB-setup-06fe71c519ac01ca8b7a3bb24cb4ca1cf8cf9a6f71de0bdc21ec247155d0ab56-3.0.2` even with this change (and thus not properly setting up the openhab instance with the changes in openhab.rake). did I specify the additional file to include in the cache key hash wrong?

Reply:
doh, never mind, I see. I need to change the hash both when the setup is cached, and when it's restored